flipt-java/src/main/java/io/flipt/api/FliptClient.java

```diff
@@ -22,27 +24,39 @@ private FliptClient(FliptClientBuilder builder) {
             .headers(builder.headers)
             .setUnhandledExceptionProcessor(builder.unhandledExceptionProcessor)
             .build();
+    this.flag =
+        Flag.builder()
+            .httpClient(httpClient)
+            .baseURL(builder.baseURL)
+            .authenticationStrategy(builder.authenticationStrategy)
+            .headers(builder.headers)
+            .setUnhandledExceptionProcessor(builder.unhandledExceptionProcessor)
+            .build();
   }
 
   public Evaluation evaluation() {
     return evaluation;
   }
 
+  public Flag flag() {
```

Comment:
would it be weird to rename this method `flags` like we have in the python SDK? https://github.com/flipt-io/flipt-server-sdks/tree/main/flipt-python/flipt/flags

Reply:
Hi @markphelps, fixed your the suggestions